src/Core/src/Exception/Http/HttpExceptionTrait.php

```diff
@@ -85,6 +85,9 @@ public function __construct(ResponseInterface $response)
                 $this->awsMessage = $body['Message'];
                 $message .= "\n\n" . $body['Message'] . "\n\n";
             }
+            if (isset($body['__type'])) {
+                list(, $this->awsCode) = explode('#', $body['__type'], 2);
```

Comment:
SDK uses `$data['code'] = isset($parts[1]) ? $parts[1] : $parts[0];` meaning, it fallback to `parts[0]`

Reply:
Done.